include/types.hpp

```diff
@@ -46,10 +46,14 @@ using IPZKwdValueMap = std::unordered_map<std::string, std::string>;
 /*IPZ VPD Map of format <Record name, <keyword, value>>*/
 using IPZVpdMap = std::unordered_map<std::string, IPZKwdValueMap>;
 
-/*Value types supported by Keyword VPD*/
-using KWdVPDValueType = std::variant<BinaryVector,std::string, size_t>;
+using MultiDataType = std::variant<BinaryVector,std::string, size_t>;
+using KeywordDataMap = std::unordered_map<std::string, MultiDataType>;
+
 /* This hold map of parsed data of keyword VPD type*/
-using KeywordVpdMap = std::unordered_map<std::string, KWdVPDValueType>;
+using KeywordVpdMap = KeywordDataMap;
+
+/* This hold map of parsed data of DDIMM VPD type*/
+using DdimmVpdMap = KeywordDataMap;
```

Comment:
DdimmVpdMap - This becomes an unused type in this case. We will need a better solution.

Reply:
nit: just place it above line 82 and have a comment there so that it should not be misleading later on.

Reply:
done

Reply:
ok, but I wanted you to add comments explaining why the type has been defined this way.

Reply:
done

Reply:
/*
 * Both Keyword VPD parser and DDIMM parser stores the
 * parsed VPD in the same format of type KeywordDataMap <Record name, variant<...>>
 * To have better readability, two types are defined for underneath data structure.
*/
using DdimmVpdMap = KeywordVpdMap;